From: Thomas Haller <thaller@redhat.com>
Date: Fri, 13 May 2016 14:19:59 +0200
Subject: libnm: don't require initialized @out_encrypted argument in
 nm_utils_file_is_private_key()

No need to require the caller to initialize the optional out-argument.
Otherwise we get:

  (nm-connection-editor:2471): libnm-CRITICAL **: nm_utils_file_is_private_key: assertion 'out_encrypted == NULL || *out_encrypted == FALSE' failed

https://bugzilla.gnome.org/show_bug.cgi?id=763578

Fixes: 1c4f41c61013d79a32eeac246d204c2f378e6915
---
 libnm-core/nm-utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libnm-core/nm-utils.c b/libnm-core/nm-utils.c
index 70c268f..1080d9e 100644
--- a/libnm-core/nm-utils.c
+++ b/libnm-core/nm-utils.c
@@ -2399,8 +2399,8 @@ nm_utils_file_is_private_key (const char *filename, gboolean *out_encrypted)
 	const char *extensions[] = { ".der", ".pem", ".p12", ".key", NULL };
 
 	g_return_val_if_fail (filename != NULL, FALSE);
-	g_return_val_if_fail (out_encrypted == NULL || *out_encrypted == FALSE, FALSE);
 
+	NM_SET_OUT (out_encrypted, FALSE);
 	if (!file_has_extension (filename, extensions))
 		return FALSE;
 
